composables/transaction/mintToken/transactionMintRmrk.ts

```diff
@@ -153,18 +149,15 @@ export async function execMintRmrk({
 
   const nameInNotifications = getNameInNotifications(item)
 
-  const isSingle = args.length === 1
-  const cb = isSingle ? api.tx.system.remark : api.tx.utility.batchAll
+  const cb = api.tx.utility.batchAll
 
-  const arg = isSingle
-    ? args
-    : [args.filter(Boolean).map((arg) => asSystemRemark(api, arg as string))]
+  const arg = [args]
 
   executeTransaction({
     cb,
     arg,
     successMessage:
-      item.successMessage ||
+      item.successMessage ??
```

Comment:
we have `item.successMessage ||` on other transaction file, should we change it there too?

Reply:
I was just following sonar lint "requests"
if you want ill open another PR to deal with that, so we will avoid mixing topics together